.github/ISSUE_TEMPLATE/feature-request.md

```diff
@@ -0,0 +1,25 @@
+---
+name: Feature Request
+about: Suggest a new functionality or data change.
+title: ""
+labels: "user request"
+assignees: ""
+---
+
+## Description
+
+Provide a detailed description of the request
```

Comment:
@atolopko-czi I'm assuming using quotes here will take care of the automated addition of the label "user request"